src/Tizen/Avalonia.Tizen/TizenThreadingInterface.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Avalonia.Platform;
+using Avalonia.Threading;
+
+namespace Avalonia.Tizen;
+internal class TizenThreadingInterface : IPlatformThreadingInterface
+{
+    internal event Action? TickExecuted;
+    private bool _signaled;
+
+    internal static SynchronizationContext MainloopContext { get; set; }
+
+    public IDisposable StartTimer(DispatcherPriority priority, TimeSpan interval, Action tick)
+    {
+        return new Timer(_ =>
+        {
+            EnsureInvokeOnMainThread(tick);
+        }, null, interval, interval);
+    }
+
+    private void EnsureInvokeOnMainThread(Action action)
+    {
+        if (SynchronizationContext.Current != null)
+            action();
+        else
+            MainloopContext.Post(_ => action(), null);
```

Comment:
I feel like this change would make sense it this place of code:
```suggestion
            MainloopContext.Post(static arg => ((Action)arg!).Invoke(), action);
```

Reply:
To avoid allocations on this branch